### PR TITLE
dotCMS/core#18293 PP Filters: Allow filter for donwload bundles

### PR DESCRIPTION
**Pull request by hmoreras**

### CODE REVIEW

File: src/app/api/services/dot-download-bundle-dialog/dot-download-bundle-dialog.service.spec.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    let service: DotDownloadBundleDialogService;
+    let bundleID: string;
+
+    beforeEach(() => {
+        TestBed.configureTestingModule({});
+        service = TestBed.get(DotDownloadBundleDialogService);
+        service.showDialog$.subscribe((data: string) => (bundleID = data));
+    });
+
+    it('should be created', () => {
```
Comment (fmontes) [inline src/app/api/services/dot-download-bundle-dialog/dot-download-bundle-dialog.service.spec.ts:14]: should what?

File: src/app/api/services/dot-download-bundle-dialog/dot-download-bundle-dialog.service.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@angular/core';
+import { Observable, Subject } from 'rxjs';
+
+@Injectable({
+    providedIn: 'root'
```
Comment (fmontes) [inline src/app/api/services/dot-download-bundle-dialog/dot-download-bundle-dialog.service.ts:5]: root?

File: src/app/view/components/main-legacy/main-legacy.component.html [hunk 3]
```diff
@@ -20,3 +20,4 @@
 <dot-large-message-display></dot-large-message-display>
 <dot-alert-confirm></dot-alert-confirm>
 <dot-push-publish-dialog></dot-push-publish-dialog>
+<dot-dot-download-bundle-dialog></dot-dot-download-bundle-dialog>
```
Comment (fmontes) [inline src/app/view/components/main-legacy/main-legacy.component.html:23]: We need to find a way to strop adding things here, we need to load this on the fly.

File: src/app/api/services/dot-download-bundle-dialog/dot-download-bundle-dialog.service.spec.ts (hunk 1 above)
Comment (hmoreras) [reply src/app/api/services/dot-download-bundle-dialog/dot-download-bundle-dialog.service.spec.ts:14]: done.

File: src/app/api/services/dot-download-bundle-dialog/dot-download-bundle-dialog.service.ts (hunk 2 above)
Comment (hmoreras) [reply src/app/api/services/dot-download-bundle-dialog/dot-download-bundle-dialog.service.ts:5]: not needed.

**Pull request merged**